src/gui/owncloudgui.cpp

```diff
@@ -89,6 +89,16 @@ ownCloudGui::ownCloudGui(Application *parent)
 
     connect(_tray.data(), &Systray::shutdown,
         this, &ownCloudGui::slotShutdown);
+    connect(_tray.data(), &Systray::openShareDialog,
+        this, [=](const QString &sharePath, const QString &localPath, const bool publicLink = false)
```

Comment:
Better just bool than const bool. Also I'm not sure what the default value is for since either openShareDialog has only two parameters and then publicLink will be always false, or openShareDialog has three parameters and the default value will be discarded.

Reply:
This is due to the fact, that the tray window only (statically) use the _false_ case (i.e. not public as default. I decided to implement the connection with the traditional signature but with a false default so that one could implement this later, if this feature is requested. I could totally omit this here, if this is your intention. 

I'll remove the const dec.

Reply:
Well, it it's not used I'd just remove the case. I live by YAGNI. ;-)

IOW: no need to introduce extra complexity by unused code.

Reply:
Removed.